massa-client/src/repl.rs

```diff
@@ -94,161 +86,107 @@ fn group_parameters(parameters: Vec<String>) -> Vec<String> {
     new_parameters
 }
 
+#[derive(Helper, Completer, Hinter, Validator, Highlighter)]
+struct MyHelper {
+    #[rustyline(Completer)]
+    completer: MassaCompleter,
+    #[rustyline(Validator)]
+    validator: MatchingBracketValidator,
+}
+
 pub(crate) async fn run(client: &Client, wallet: &mut Wallet) {
     massa_fancy_ascii_art_logo!();
-    println!("Use 'exit' to quit the prompt");
+    println!("Use 'CTRL+D or CTRL+C' to quit the prompt");
     println!("Use the Up/Down arrows to scroll through history");
     println!("Use the Right arrow or Tab to complete your command");
     println!("Use the Enter key to execute your command");
     crate::cmds::help();
-    let mut history = CommandHistory::default();
-    let completion = CommandCompletion::default();
+    let h = MyHelper {
+        completer: MassaCompleter::new(),
+        validator: MatchingBracketValidator::new(),
+    };
+    let config = Config::builder()
+        .auto_add_history(true)
+        .completion_prompt_limit(100)
+        .completion_type(CompletionType::List)
+        .max_history_size(10000)
+        .build();
+    let mut rl: Editor<MyHelper> = Editor::with_config(config).unwrap();
```

Comment:
I'm not a big fan of unwrap(). Could be replaced with `?`or handle the error explicitly